sample/AspNetFullFrameworkSampleApp/MyModule1.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Elastic.Apm.DiagnosticSource;
+using System;
+using System.Web;
+using Elastic.Apm;
+
+namespace AspNetFullFrameworkSampleApp
+{
+	public class MyModule1 : IHttpModule
+	{
+		private IDisposable _httpDiagnosticsSubscription;
+
+		/// <summary>
+		/// You will need to configure this module in the Web.config file of your
+		/// web and register it with IIS before being able to use it. For more information
+		/// see the following link: https://go.microsoft.com/?linkid=8101007
+		/// </summary>
+
+		#region IHttpModule Members
+
+		public void Dispose() => _httpDiagnosticsSubscription.Dispose();
+
+		public void Init(HttpApplication context)
+		{
+			//start listening for outgoing HTTP requests in the Elastic APM Agent.
```

Comment:
Aren't those comments outdated? I think those can be removed.

Reply:
Done